subprojects/android-test/test-app/build.gradle.kts

```diff
@@ -178,6 +178,33 @@ extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
             }
         }
     }
+
+    configurationsContainer.register("uiDebug") {
+        tryToReRunOnTargetBranch = false
+        reportSkippedTests = false
+        rerunFailedTests = false
+        reportFlakyTests = false
+        // uncomment after 2020.3.6 release (MBS-8050)
+        // debugMode = true
+
+        targetsContainer.register("api27") {
+            deviceName = "API27"
+
+            scheduling = SchedulingConfiguration().apply {
+                quota = QuotaConfiguration().apply {
+                    retryCount = 1
+                    minimumSuccessCount = 1
+                }
+
+                reservation = TestsBasedDevicesReservationConfiguration().apply {
```

Comment:
All blockers for usage must be explicit. Not it's not a blocker to merge but blocker of DoD.
Create a task for describing debug restriction or make them clear for clients.

Reply:
About what blockers are you talking about?

Reply:
Discussed in private chat.